src/module/apps/trait-selectors/weapon-property-selector.js

```diff
@@ -0,0 +1,75 @@
+import { TraitSelectorSFRPG } from "../trait-selector.js";
+
+export class WeaponPropertySelectorSFRPG extends TraitSelectorSFRPG {
+    constructor(item, options) {
+        super(item, options);
+        super.getData();
```

Comment:
Don't need to getData here, it's called during `render` anyway

Reply:
Removed